gravitee-apim-console-webui/src/management/api/runtime-logs-v4/runtime-logs/components/api-runtime-logs-quick-filters/components/api-runtime-logs-more-filters/components/api-runtime-logs-more-filters-form.component.ts

```diff
@@ -33,13 +33,13 @@ export class ApiRuntimeLogsMoreFiltersFormComponent implements OnInit, OnDestroy
   @Output() valuesChangeEvent: EventEmitter<MoreFiltersForm> = new EventEmitter<MoreFiltersForm>();
   @Output() isInvalidEvent: EventEmitter<boolean> = new EventEmitter<boolean>();
   readonly periods = PERIODS;
-  moreFiltersForm: FormGroup;
+  datesForm: FormGroup;
```

Comment:
Here we use a dedicated form for the dates because in the next commit will reintroduce the moreFiltersForm for all the other form fields. I mainly did it to be able to subscribe to dates form controls like it's done in the onDatesChange method. On the other hand we will subscribe to all the other form controls values and emit them like they are.